client/src/components/Conversations/Conversations.tsx

```diff
@@ -2,6 +2,54 @@ import Convo from './Convo';
 import Conversation from './Conversation';
 import { useLocation } from 'react-router-dom';
 import { TConversation } from 'librechat-data-provider';
+import { parseISO, isToday, isWithinInterval, subDays, getYear } from 'date-fns';
+
+const getGroupName = (date) => {
+  const now = new Date();
+  if (isToday(date)) {
+    return 'Today';
+  }
+  if (isWithinInterval(date, { start: subDays(now, 7), end: now })) {
+    return 'Last 7 days';
+  }
+  if (isWithinInterval(date, { start: subDays(now, 30), end: now })) {
+    return 'Last 30 days';
+  }
+  return ' ' + getYear(date).toString(); // Returns the year for anything older than 30 days
+};
+
+// Function to group conversations
+const groupConversationsByDate = (conversations) => {
+  const groups = conversations.reduce((acc, conversation) => {
+    const date = parseISO(conversation.createdAt);
```

Comment:
Use the `updatedAt` field, this is more in line with the official version

Reply:
Right.